includes/class-wc-payments-utils.php

```diff
@@ -202,7 +202,6 @@ public static function zero_decimal_currencies(): array {
 			'mga', // Malagasy Ariary.
 			'pyg', // Paraguayan Guaraní.
 			'rwf', // Rwandan Franc.
-			'ugx', // Ugandan Shilling.
```

Comment:
Not sure if any changes needs to be made here https://github.com/Automattic/woocommerce-payments/blob/e6e7c3be3d06153233bd14159bb2c4f0f1e120b4/i18n/locale-info.php#L3667

Reply:
I think not since it's a Stripe custom rule for this specific currency.